python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -192,7 +192,23 @@
 data_train_scaled.describe()
 
 # %% [markdown]
-# We can easily combine these sequential operations with a scikit-learn
+# We can use a jointplot to visualize the histograms and scatterplot of any
+# pair of numerical features at the same time. We can observe that `StandardScaler`
+# does not change the structure of the data itself but the axes
+# get shifted and scaled.
+
+# %%
+import matplotlib.pyplot  as plt
+import seaborn as sns
+
+sns.jointplot(data=data_train[:300], x="age", y="hours-per-week", marginal_kws=dict(bins=15))
```

Comment:
Put a comment before saying where the magical 300 comes from something like this:

```py

# number of points to visualize to have a clearer plot
num_points_to_plot = 300

sns.jointplot(data=data_train[:num_points_to_plot], ...)

```